docs/web-configuration.md

```diff
@@ -86,8 +86,8 @@ which will look something like:
 `$2y$10$X0h1gDsPszWURQaxFh.zoubFi6DXncSjhoQNJgRrnGs7EsimhC7zG`
 
 The cost (10 in the example) influences the time it takes for computing the
-hash. A higher cost will en up slowing down the authentication process.
-Depending on the machine, a cost of 10 will take about ~70ms where a cost of
+hash. A higher cost will end up slowing down the authentication process.
+Depending on the machine, a cost of 10 will take about ~70ms, whereas a cost of
 18 can take up to a few seconds. That hash will be computed on every
```

Comment:
```suggestion
18 can take up to a few seconds. That hash will be computed on the first authenticated HTTP request and then cached.
```

Reply:
Sure, I just force-pushed with that change.

Reply:
I do not see the commit

Reply:
Problem was between keyboard and chair ;) should be there now!